noark-extraction-validator/src/main/java/com/documaster/validator/storage/database/DatabaseStorage.java

```diff
@@ -214,6 +214,9 @@ public void destroy() {
 
 	private void createSchema(String schemaName) throws SQLException {
 
+		// Required when running multiple times using HSQLDB_FILE to delete the data from the previous run
+		conn.createStatement().execute(MessageFormat.format("DROP SCHEMA IF EXISTS {0} CASCADE;", schemaName));
```

Comment:
Shouldn't we delete the file at the end of the validation instead? For large extractions it may become pretty big and the user might not want to have GBs worth of validator temp files present.

What do you think?

Reply:
Fixed in: https://github.com/documaster/noark-extraction-validator/pull/52/commits/29091f39fc94ad042f8b02ee7eed104b1a3643d7